src/pages/finalize.js

```diff
@@ -173,10 +174,17 @@ export const finalizeTemplate = (data, specimenData, bptlCollectionFlag) => {
         openModal(modalId);
     });
 
-    document.getElementById('finalizedConfirmButton') && document.getElementById('finalizedConfirmButton').addEventListener('click', async (e) => { 
-        specimenData['410912345'] = 353358909;
-        specimenData['556788178'] = new Date().toISOString();
+    document.getElementById('finalizedConfirmButton') && document.getElementById('finalizedConfirmButton').addEventListener('click', async () => { 
+        try {
+        specimenData[conceptIds.collection.isFinalized] = conceptIds.yes;
```

Comment:
incredibly small, but please indent code in this try block

Reply:
ahhh missed it. Updated, thanks.